tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -54,6 +55,9 @@ services:
       - service-mesh
     working_dir: "/awx_devel"
     volumes:
+      - type: bind
+        source: /run/user/1000/keyring/ssh
```

Comment:
this wont work. see https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#interpolation-syntax